src/services/shipment_service.js

```diff
@@ -205,7 +205,11 @@ export default (easypostClient) =>
      */
     static async lowestSmartRate(id, deliveryDays, deliveryAccuracy) {
       const smartRates = await this.getSmartRates(id);
-      return Util.getLowestSmartRate(smartRates, deliveryDays, deliveryAccuracy.toLowerCase());
+      return new EasyPostClient('fake_api_key').Utils.getLowestSmartRate(
```

Comment:
issue: we should not be constructing clients inside our own code. The client is for the user. Because we have direct access to the code before it gets compiled, we should still be able to reference the utils relatively. It's our users who cannot do this because they only get access to the single exposed client.